content/26.acknowledgments.md

```diff
@@ -0,0 +1,21 @@
+## Acknowledgments
+
+We would like to thank the Gordon and Betty Moore Foundation whose support makes the study possible (GBMF4552 to D.S.H. and GBMF4552 to C.S.G.). 
+We thank the reviewers and editors for their insightful input to improve the quality of the study.
+
+## Author Contributions
+
+Conceptualization, C.S.G.;
+Methodology, T.T.L., D.S.H., A.A.H., M.R.G. and C.S.G.;
+Software, T.T.L., D.S.H., A.A.H. and M.R.G.;
+Investigation T.T.L., D.S.H., A.A.H., M.R.G. and C.S.G.;
+Data Curation, D.S.H., A.A.H. and M.R.G.;
```

Comment:
```suggestion
Data Curation, D.S.H., A.A.H., M.R.G. and C.S.G.;
```
The rare project where I curated things (Initial honoree table)!

Reply:
Ah true. Sorry I forgot about that. I contributed to that, too! I'll add.